include/deal.II/base/multithread_info.h

```diff
@@ -55,13 +55,11 @@ class MultithreadInfo
   MultithreadInfo() = delete;
 
   /**
-   * The number of CPUs in the system. At the moment detection of CPUs is only
-   * implemented on Linux, FreeBSD, and Mac computers.  It is one if detection
-   * failed or is not implemented on your system.
+   * The number of CPUs in the system.
    *
-   * If it is one, although you are on a multi-processor machine, please refer
-   * to the documentation in <tt>multithread_info.cc</tt> near to the
-   * <tt>error</tt> directive.
+   * This internally calls
+   * [<code>std::thread::hardware_concurrency</code>](https://en.cppreference.com/w/cpp/thread/thread/hardware_concurrency)
```

Comment:
does doxygen understand this format?

Reply:
Yes. See the links to `std::future` in the "Detailed description" here: https://dealii.org/developer/doxygen/deal.II/classThreads_1_1Task.html